operator/README.md

```diff
@@ -0,0 +1,5 @@
+# Operator
```

Comment:
I think we can rename this to `Datadog Operator` since we already have precedence with the [Datadog Cluster Agent](https://github.com/DataDog/integrations-core/tree/master/datadog_cluster_agent). Can you try changing the root directory to `datadog_operator` and updating all references from "Operator" to "Datadog Operator"?